### PR TITLE
Added Adrenaline Syringe

### PR DESCRIPTION
**Pull request by bleehu**
modified:   07_Items.txt

### CODE REVIEW

File: Items/07_Items.txt [hunk 1]
```diff
@@ -128,6 +128,19 @@ wearing any helmets or other sizeable headgear.
...
 
+Adrenaline Syringe ($18):
+    
+    This small metal cylinder contains a few ounces of synthetic adrenaline.
+This particular dose doesn't last long, but will keep your heart beating even if
+it has almost no blood left in it at all. 
+    
+    Using this syringe takes two Actions. The syringe can only be used on an 
+organic character who is bleeding out. The syringe does not heal the character 
+or stabalize them. The character's Death Saving Throws are all reset as if that
```
Comment (Turtlelord26) [inline Items/07_Items.txt]: Sp: stabilize

**Review by Turtlelord26 — CHANGES_REQUESTED** [review]
NB: It feels more expensive than 18, probably in part due to the flavor
> the device measures out the appropriate dosage automatically

I don't see any obvious mechanical or balance issues with such an item existing, and from a level design perspective it adds a cool small-loot option. 👍 

Flavor police incoming: Does the adrenaline work on Security Drones too? How about Kurish? Approval pending this flavor point discussion only.

**Comment by bleehu** [comment]
Line 138 specifically says organic character, so no Security Drones. Kurish is up for debate.

**Comment by Turtlelord26** [comment]
My bad.

**Review by Turtlelord26 — APPROVED** [review]

**Comment by Turtlelord26** [comment]
We can decide later if realism or gameplay is more important re having a different adrenaline for each species. I have a hunch which way that has to go.

**Review by rosebrallier — APPROVED** [review]
I like it! Ship pending spelling fixes.

**Pull request merged**